### PR TITLE
Fix/aut 341/html injection rubric block

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
**Related to**
[AUT-341](https://oat-sa.atlassian.net/browse/AUT-341)

**How to test**
1. Login as admin
2. Create test
3. Add new Rubric Block
4. Inject script code test payload into the text label content input field
5. Save the entry and click on feedback -> activate
6. Injected code should not be executed

### CODE REVIEW

File: views/js/controller/creator/encoders/dom2qti.js [hunk 1]
```diff
@@ -137,40 +138,42 @@ define([
...
 
             _.forEach($nodeValue, function (elt) {
-                var object;
+                let object;
                 if (elt.nodeType === 3) {
                     if (!_.isEmpty($.trim(elt.nodeValue))) {
                         result.push(qtiElementHelper.create('textRun', {
-                            'content': elt.nodeValue,
+                            'content': DOMPurify.sanitize(elt.nodeValue, {
+                                FORBID_TAGS: ['script']
```
Comment (bziondik) [inline views/js/controller/creator/encoders/dom2qti.js:152]: Do we need `FORBID_TAGS: ['script']`?
I think `script` is forbidden by default

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: views/js/controller/creator/encoders/dom2qti.js (hunk 1 above)
Comment (AnastasiaArcadia) [reply views/js/controller/creator/encoders/dom2qti.js]: Yes, you are right

**Comment by 404passionFruit** [comment]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by tereshhhchenko — APPROVED** [review]

**Pull request merged**